app/models/planets.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy.orm import Mapped, mapped_column
+from ..db import db
+
+class Planet(db.Model):
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    name: Mapped[str]
+    description: Mapped[str]
+    size: Mapped[int]
+
+    def to_dict(self):
+        return dict(
+            id=self.id,
+            name=self.name,
+            description=self.description,
+            size=self.size
+        )
+
```

Comment:
In terms of refactoring, adding a `from_dict` class method could be super useful!